enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/server/AuthServer.java

```diff
@@ -137,6 +138,9 @@ public static void main(String[] args) {
             server.start();
             logger.info("Sever started Listening in port : " + 8081);
 
+            // Initialize JMX Agent
+            JMXAgent.initJMXAgent();
```

Comment:
are we initializing this all the time?? shouldn't it be done based on a config?

Reply:
It's getting checked inside the `initJMXAgent` method.
```
public static void initJMXAgent() {
        if (JMXUtils.isJMXMetricsEnabled()) {
            try {...
```